src/parser_complex_test.cc

```diff
@@ -9,15 +9,13 @@
 
 #if defined(_MSC_VER)
 #define timegm _mkgmtime
-#endif
-
-using namespace std;
-
+#elif defined(_WIN32)
```

Comment:
I am actually not entirely certain what this is doing since it seems to work just fine If I comment it out the #define or leave it in.  Not sure if it is a MSC_VER issue so it is needed in older versions.   What I know if both the `#define timegm and using toml::timegm are active it doesn't compile.

Reply:
IIRC, code around here is for cygwin environment.